package.json

```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "start": "nodemon server.js",
-    "test": "jest",
+    "test": "mocha tests/**/*.test.js",
```

Comment:
Because we haven't begun using Jest yet, I'm personally fine with moving to Mocha. The rest of the team should have a say on this decision though.

Reply:
And in case the move to mocha was made to attempt to get the build to succeed, I think we can just revert the change.

Reply:
Yeah I think unless there is a good reason to use mocha, we should stick to jest.